src/nghttp.cc

```diff
@@ -687,7 +687,9 @@ void HttpClient::disconnect() {
   state = ClientState::IDLE;
 
   for (auto req = std::begin(reqvec); req != std::end(reqvec); ++req) {
-    (*req)->continue_timer->stop();
+    if ((*req)->continue_timer) {
```

Comment:
Go would be a new language for me, but if you'd prefer to have tests on this feature before pulling it in, I can see what I can do. No promises on how quickly it would happen, of course. :grinning: